nifi-registry-core/nifi-registry-web-api/pom.xml

```diff
@@ -445,5 +445,11 @@
             <version>${jetty.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi.registry</groupId>
+            <artifactId>nifi-registry-jetty</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
We shouldn't have to add this since the web-api uses Spring Security's configuration and doesn't use our custom filters right?

Reply:
Yep I'll remove that